rhoas-client/src/main/java/com/redhat/service/bridge/rhoas/RhoasClient.java

```diff
@@ -0,0 +1,12 @@
+package com.redhat.service.bridge.rhoas;
+
+import com.redhat.service.bridge.rhoas.dto.TopicAndServiceAccountRequest;
+import com.redhat.service.bridge.rhoas.dto.TopicAndServiceAccountResponse;
+
+import io.smallrye.mutiny.Uni;
+
+public interface RhoasClient {
+
+    Uni<TopicAndServiceAccountResponse> createTopicAndConsumerServiceAccount(TopicAndServiceAccountRequest request);
```

Comment:
@kostola I don't see this integrated into any of the code that is called when the user requests an OpenBridge Instance? How/When is this method actually being called?

Reply:
I integrated it in the manager inside `BridgesServiceImpl`. Take a look :nerd_face: